languages/go/exercises/concept/comments/.docs/after.md

```diff
@@ -0,0 +1,67 @@
+In the previous exercise, we saw that there are two ways to write comments in Go: single-line comments that are preceded by `//`, and multiline comment blocks that are wrapped with `/*` and `*/`.
+
+## Comments for documentation
+
+In Go, comments play an important role in documenting code. They are used by the [godoc][godoc] command, which extracts these comments to create documentation about Go packages (see [pkg.go.dev][go packages] for examples). A documentation comment should be a complete sentence that starts with the name of the thing being described and ends with a period.
+
+For comments other than documentation comments, while they may be helpful for maintainers, a [good rule of thumb][less comments] and more sustainable solution is to write code that is easier to understand so that explanatory comments are hopefully not needed.
+
+## Comments for exported identifiers
+
+Documentation comments should precede packages as well as [exported identifier][exported identifiers]s, for example exported functions, methods, package variables, constants, and structs, which you will learn more about in the next exercises. Comments written for packages and exported identifiers are useful for the users who import and use these packages.
+
+Note, however, that identifiers (such as variables) that are declared inside of functions and methods are private and do not necessarily require comments for the users of the packages.
+
+A package-level variable can look like this:
+
+```go
+// TemperatureFahrenheit gives a certain temperature in degrees Fahrenheit.
+var TemperatureFahrenheit float64
+```
+
+Note that `TemperatureFahrenheit` is capitalized, which makes this identifier exported, and thus usable in any code which imports this package.
+
+## Package comments
+
+Package comments should be written directly before a package clause (`package x`) and begin with `Package X ...` like this:
+
+```go
+// Package kelvin provides tools to convert temperatures to and from Kelvin.
+package kelvin
+```
+
+## Function comments
+
+A function comment should be written directly before the function declaration. It should be a full sentence that starts with the function name. For example, an exported comment for `function Test` should take the form `Test ...`. It should also explain what arguments the function takes, what it does with them, and what its return values mean, ending in a period):
```

Comment:
Maybe:

```suggestion
A function comment should be written directly before the function declaration. It should be a full sentence that starts with the function name. For example, an exported comment for the function `Test` should take the form `Test ...` ending in a period. It could also explain what arguments the function takes and what its return values mean:
```